### PR TITLE
Time functions

### PR DESCRIPTION
**Pull request by ppease**
### What does this Pull Request accomplish?

Add several methods related to time functionality for the time reference test panel for NI-Sync.
Methods Added:
```
SetTime
GetTime
SetTimeReferenceFreeRunning
SetTimeReferenceGPS
SetTimeReferenceIRIG
SetTimeReferencePPS
SetTimeReference1588OrdinaryClock
SetTimeReference8021AS
```
There is also a change to the nisync.lib file.  We were improperly declaring some of the function prototypes in the nibuild component that generated the stub .lib file.  I updated that, and am updating it here too.

### Why should this Pull Request be merged?

Adding methods need for the time reference test panel for NI-Sync.

### What testing has been done?

System tests were added for all new methods, and they are passing on a windows system with a 6683.  I am currently disabling these new tests because we don't have any logic to determine if a 6683 is available and these tests will fail with a 6674. I have another task that handles removing the disabled flags from all the tests.

### CODE REVIEW

File: source/codegen/proto_helpers.py [hunk 1]
```diff
@@ -38,6 +38,8 @@ def get_grpc_type_from_ivi(type, is_array, driver_name_pascal):
     type = 'sint32'
   if 'ViInt64' in type:
     type = 'int64'
+  if 'ViUInt16' in type:
```
Comment (ppease) [inline source/codegen/proto_helpers.py:41]: I guess I was the first one to use ViUInt16 in our API.  I looked through the documentation for the proto standard, and it looks like there isn't a representation for 16 bit numbers.  Above this we are already turning ViInt8 into a 32 bit integer so I am doing the same here for the 16 bit type.

**Review by zhindes — CHANGES_REQUESTED** [review]

File: source/tests/system/nisync_driver_api_tests.cpp [hunk 2]
```diff
@@ -725,6 +844,146 @@ TEST_F(NiSyncDriverApiTest, MeasureFrequencyExOnOscillatorWithFrequency_ReturnsF
   EXPECT_GT(frequency, 0);
 }
 
+TEST_F(NiSyncDriverApiTest, SetTimeWithValidTimeSource_ReturnsSuccess)
```
[the remark this answers is not in the document]
Comment (tkrebes) [reply source/tests/system/nisync_driver_api_tests.cpp:847]: The updates I'm making to the NI-Sync system tests in #192 should make that  easier to do.

[the remark this answers is not in the document]
Comment (zhindes) [reply source/tests/system/nisync_driver_api_tests.cpp:847]: Yeah, I just saw that. Awesome!

[the remark this answers is not in the document]
Comment (ppease) [reply source/tests/system/nisync_driver_api_tests.cpp]: For now I am disabling these tests.  I am creating a separate task under the same user story to address splitting up the tests based on model and removing the DISABLED_ from the tests.

**Review by zhindes — APPROVED** [review]

**Review by tkrebes — APPROVED** [review]

**Review by jonlugoatx — APPROVED** [review]

**Review by reckenro — APPROVED** [review]

**Pull request merged**